index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header>
+            <p class="time"> 9:14 <img class="net" src="C:\Users\Harrison\phone network image.jpg" alt="phone network"></p>
+            <img class="logo" src="C:\Users\Harrison\Downloads\My Logo.png" alt="logo">
+            <img class="navicon" src="https://assets.stickpng.com/images/588a64c6d06f6719692a2d0c.png" alt="nav icon"><br>
+         
+          </header>
```

Comment:
- [ ] Nice implementation wrapping this part (header) with `<header>` tag, the header tag is used as a page header, which it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info 💯.